nomad/acl_endpoint.go

```diff
@@ -11,6 +11,7 @@ import (
 	metrics "github.com/armon/go-metrics"
 	log "github.com/hashicorp/go-hclog"
 	memdb "github.com/hashicorp/go-memdb"
+	policy "github.com/hashicorp/nomad/acl"
```

Comment:
I'd recommend not shadowing the name here, when reading the code it can make interpretation difficult when there are multiple names for one package.

Reply:
my previous attempt at this had it called `somethingacl` because whenever I didn’t have an override, this whole line just got deleted! I’m guessing because `acl` is already defined in the document…????

Reply:
Ah - I guess so :( - That's unfortunate but makes sense :+1: